src/Compilers/CSharp/Portable/BoundTree/BoundImplicitIndexerAccess.cs

```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Diagnostics;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
```

Comment:
It doesn't look like there are meaningful changes in this file. Consider reverting.